petpals/messaging/static/js/conversation_message.js

```diff
@@ -0,0 +1,91 @@
+var submitButton = document.getElementById('submit');
+
+current_user = document.getElementById("conversation").dataset['current_user']
+
+
+recipient = document.getElementById("conversation").dataset["recipient"];
+sender = document.getElementById("conversation").dataset["sender"];
+message = document.getElementById("new_message").value.replaceAll("`", "");
+conversation_id = document.getElementById("conversation").dataset["conversation_id"]
+
+document.getElementById("message-container").scrollTop = document.getElementById("message-container").scrollHeight;
+
+// Socket IO
+socket = io()
+socket.connect('http://127.0.0.1:5000/messages/conversation/' + conversation_id)
```

Comment:
Should be a relative address.

Reply:
Socket IO, for some reason, requires an entire webaddress.